website/src/components/NavBarItem.js

```diff
@@ -1,10 +1,23 @@
 import React from 'react'
 import { Button, Link } from '@mui/material';
+import { NavLink, useMatch } from 'react-router-dom';
 
 export default function NavBarItem({ href, children }){
+
+    const match = useMatch(href);
     return (
-        <Link href={href} color='inherit' underline='none'>
-            <Button color='inherit'>{children}</Button>
+        <Link component={NavLink} to={href} color='inherit' underline='none'>
```

Comment:
Styling looks cool, but lets keep it more basic -- bold and/or a muted text color should do the trick

Reply:
Definitely reference MUI for this if you can

Reply:
![image](https://github.com/Connor-Bernard/gradeView/assets/48500458/2efd7190-f087-4377-8ae8-aa4a9bcbe571)
New underlining using Material UI's Box component